custom_components/opnsense/switch.py

```
@@ -10,241 +11,292 @@
+                    continue
+                icon = "mdi:network"
+                # likely only want very specific rules to manipulate from actions
                 enabled_default = False
                 # entity_category = ENTITY_CATEGORY_CONFIG
                 device_class = SwitchDeviceClass.SWITCH
+                tracker = dict_get(rule, "created.time")
+                # we use tracker as the unique id
+                if tracker is None or len(tracker) < 1:
+                    continue
+
+                if "descr" not in rule.keys():
```

Ruff

Use `key not in dict` instead of `key not in dict.keys()`

Remove `.keys()`

(SIM118)